nav2_gps_waypoint_follower_demo/params/city_world_gps_waypoints.yaml

```diff
@@ -0,0 +1,26 @@
+gps_waypoint_follower_demo:
+  ros__parameters:
+    number_of_gps_waypoints: 21
+    #long, lat, alt
+    gps_waypoint0: [9.677703999088216e-07, -5.306676831178058e-05, 0.6442248001694679]
```

Comment:
I'm not super in love with how these are set in parameter file. Eventually I think this  tutorial will be updated for semantic info and use that format, but that might be a ways off from now.

I think the best thing to do here is to make it analog to the other settings we have in Nav2. e.g.
```
ros__parameters:
  waypoints: [wp0, wp1, wp2, ..., wpN]
  wp0: [43. 4332. 432]
  wp1: [3432 4323 4324]
  ...
  wpN: [ffd ffs fdfs]

```

So you parse the `waypoints` parameter as a vector of strings, then for each string in the vector, get the parameter value that is assigned to it. That lets you define different order of waypoints without renaming them and no need to pre-define the total number of waypoints anywhere.  

(you can still use `gps_waypointN`, I just short handed `wp0` for brevity)